scripts/gpu-box-setup/roles/common/handlers/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: reload sysctl
+  command: sysctl -p
+  changed_when: false
```

Comment:
Will all the systems have `systemd`?

Reply:
the dockerized systems, no. but this is not meant for the dockerized system (eg like celium). it's meant for bare-metal ubuntu. If that's the case, it's expected to have nowadays. I shall added in the requirements in the docs. thanks!